### PR TITLE
🔖 Release 1.0.2

### PR DESCRIPTION
**Pull request by tjarbo**
As Dependabot has submitted many dependency updates in the last time, we should release a new version to update the docker images.

## Changelog

* ⬆️ Bump docker/metadata-action from 3.4.0 to 3.4.1 (#153) @dependabot
* ⬆️ Bump docker/metadata-action from 3.3.0 to 3.4.0 (#150) @dependabot
* ⬆️ Bump actions/setup-node from 2.1.5 to 2.2.0 (#148) @dependabot
* ⬆️ Bump normalize-url from 4.5.0 to 4.5.1 in /packages/backend (#140) @dependabot
* ⬆️ Bump glob-parent from 5.1.1 to 5.1.2 in /packages/backend (#141) @dependabot
* ⬆️ Bump postcss from 7.0.35 to 7.0.36 in /packages/frontend (#142) @dependabot
* ⬆️ Bump ws from 6.2.1 to 6.2.2 in /packages/frontend (#139) @dependabot
* ⬆️ Bump ws from 7.3.1 to 7.4.6 in /packages/backend (#137) @dependabot
* ⬆️ Bump dns-packet from 1.3.1 to 1.3.4 in /packages/frontend (#136) @dependabot
* ⬆️ Bump browserslist from 4.12.0 to 4.16.6 in /packages/frontend (#135) @dependabot
* ⬆️ Bump docker/metadata-action from 3.2.0 to 3.3.0 (#138) @dependabot

### CODE REVIEW

**Review by antonplagemann — APPROVED** [review]
Checks and merge conflicts, everything else looks good 👍 💯

**Review by piuswalter — APPROVED** [review]

File: .github/workflows/create-issue-branch.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      issues: write
+      packages: none
+      pull-requests: write
+      repository-projects: none
+      security-events: none
+    steps:
+      - name: Create Issue Branch
+        uses: robvanderleek/create-issue-branch@main
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (p-fruck) [inline .github/workflows/create-issue-branch.yml:27]: newline!

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      issues: write
+      packages: none
+      pull-requests: write
+      repository-projects: none
+      security-events: none
+    steps:
+      - name: Run release drafter
+        uses: release-drafter/release-drafter@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (p-fruck) [inline .github/workflows/release-drafter.yml:27]: newline!

File: .github/workflows/create-issue-branch.yml (hunk 1 above)
Comment (piuswalter) [inline .github/workflows/create-issue-branch.yml:27]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

File: .github/workflows/release-drafter.yml (hunk 2 above)
Comment (piuswalter) [inline .github/workflows/release-drafter.yml:27]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

**Review by piuswalter — APPROVED** [review]

File: packages/backend/src/controllers/moodle/handle.ts [hunk 3]
```diff
@@ -24,6 +25,17 @@ export async function handleAssignments(courses: ICourse[], lastFetch: number):
         minute: 'numeric'
     };
 
+    return new Date(timestamp * 1000).toLocaleString('de-DE', dateOptions);
```
Comment (p-fruck) [inline packages/backend/src/controllers/moodle/handle.ts:28]:
```suggestion
    return new Date(timestamp * 1000).toLocaleString(config.langFormat || unknown, dateOptions);
```
Not an actual suggestion, but at one point in time we should think about i18n support for our backend

**Review by p-fruck — CHANGES_REQUESTED** [review]

File: packages/backend/src/controllers/moodle/handle.ts (hunk 3 above)
Comment (tjarbo) [reply packages/backend/src/controllers/moodle/handle.ts:28]: Sure, feel free to start #126 #127 :yum: But yeah, you are right.

**Review by p-fruck — APPROVED** [review]

**Pull request merged**